pkg/api/types.go

```diff
@@ -334,6 +441,8 @@ type Capabilities struct {
 type ResourceRequirements struct {
 	// Limits describes the maximum amount of compute resources required.
 	Limits ResourceList `json:"limits,omitempty"`
+	// Requests describes the minimum amount of compute resources required.
+	Requests ResourceList `json:"requests,omitempty"`
```

Comment:
Elsewhere in this thread @bgrant0607 suggested using ResourceRequirements w/ Requests.   I can get away with using Limits but knowing the inverse is what they are asking for (a minimum, rather than a limit).  Up to you and Brian to hash out.  I have no problem with either one.